cms/static/cms/sass/components/_pagetree.scss

```diff
@@ -405,76 +326,136 @@
     float: right;
 }
 
+.cms-tree-item-lang .cms-pagetree-dropdown-menu {
+    right: 32px;
+    top: -8px;
+    width: 180px;
+}
+.cms-tree-item-button .cms-pagetree-dropdown-menu {
+    right: 40px;
+    top: -1px;
+    width: 205px;
+}
+.cms-pagetree-alt-mode .cms-tree-item-button .cms-icon-pencil {
+    @include icon(cogs);
+}
+
 //###########################################################
-// TOOLTIP
-// handles tooltip styling
-.cms-tree-tooltip-container {
+// DROPDOWN
+// dropdowns used for menus
+.cms-pagetree-dropdown {
+    position: relative;
+}
+.cms-pagetree-dropdown-trigger {}
+
+.cms-pagetree-dropdown-menu {
     display: none;
     position: absolute;
-    right: $pagetree-tooltip-offset;
-    top: 2px;
-    z-index: 100;
-    text-align: left;
-    border-radius: $border-radius-base;
-    background: $white;
-    border: 1px solid $gray-lighter;
+    z-index: z(pagetree, dropdown);
+    right: -1px;
+    top: 30px;
+    min-width: $dropdown-width;
+    background: $submenu-dropdown-bgcolor;
+    border-radius: $border-radius-normal;
+    box-shadow: $dropdown-shadow;
+    transform: translateZ(0);
 
-    // icon on the right
-    &:before {
-        content: "";
-        position: absolute;
-        left: 100%;
-        top: $pagetree-icon-offset + 2;
-        z-index: 1;
-        width: $dropdown-arrow-side;
-        height: $dropdown-arrow-side;
-        margin-left: -($dropdown-arrow-side / 2);
-        transform: rotate(45deg);
-        box-shadow: $dropdown-shadow;
-        background-color: $submenu-dropdown-bgcolor;
+    a {
+        display: block;
+        color: $gray;
+        line-height: 1.5;
+        padding: 10px $pagetree-dropdown-padding-horizontal;
+        text-align: left;
+        &:hover,
+        &:active,
+        &:focus {
+            color: $white !important;
+            background: $color-primary;
+        }
+    }
+    &.cms-pagetree-dropdown-menu-condensed {
+        a {
```

Comment:
but why? it doesn't impede readability and actually makes the code easier to extend, no?